tests/e2e/utils/src/components.js

```diff
@@ -435,6 +435,40 @@ const createCoupon = async ( couponAmount = '5', discountType = 'Fixed cart disc
 	return couponCode;
 };
 
+/**
+ * Update an order.
+ *
+ * @param noticeText The text that appears in the notice after updating the order.
+ */
+const updateOrder = async ( noticeText ) => {
```

Comment:
Can we call this `clickUpdateOrder`? Eventually we may have functions that update orders via the api.

Reply:
I agree with that, plus it's clearer that it's clicking the update button). Updated.